.github/workflows/wheels.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           mv NOTICE_DEFAULT THIRD-PARTY-LICENSES
 
       - name: Build wheels for s3torchconnectorclient
-        uses: pypa/cibuildwheel@v2.16.2
+        uses: pypa/cibuildwheel@v2.16.5
```

Comment:
I know we were planning to do this, but is it necessary for this PR?

Reply:
No, but I tested with this as well afterwards, so I thought of including it